sql/driving_distance/drivingDistance.sql

```diff
@@ -27,7 +27,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 -- MULTIPLE
 --v3.6
-CREATE FUNCTION pgr_drivingDistance(
+CREATE FUNCTION pgr_v4drivingDistance(
```

Comment:
Any user facing function (aka, the non-underscored one) should not have `v4` prefix. All end-user facing function names should be consistent - they should be named similar to others.

The underscore functions can have any name - e.g. the `v4` prefix, because we don't expect the end users to use these functions.

Reply:
OKAY got that